cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Is this file needed?

Reply:
I didn't come across its use case specifically. Thing is, these files were automatically generated by Cypress installation process. Including `screenshot` and `video` folders. Which I have git-ignored.

I think it will be useful. I am not 100% sure at the moment about this file.

Reply:
Actually, we do need this file for some tests.. I can't remember where exactly.